pkg/config/options.go

```diff
@@ -77,14 +90,23 @@ func Init(args []string) (*Options, error) {
 
 func newOptions() *Options {
 	return &Options{
-		ProxyWebSockets:      true,
-		ListeningAddress:     ":443",
-		Elasticsearch:        "https://localhost:9200",
-		UpstreamFlush:        time.Duration(5) * time.Millisecond,
-		RequestLogging:       false,
-		AuthBackEndRoles:     map[string]BackendRoleConfig{},
-		AuthWhiteListedNames: []string{},
-		AuthAdminRole:        "",
+		ProxyWebSockets:           true,
+		ListeningAddress:          ":443",
+		Elasticsearch:             "https://localhost:9200",
+		UpstreamFlush:             time.Duration(5) * time.Millisecond,
+		RequestLogging:            false,
+		AuthBackEndRoles:          map[string]BackendRoleConfig{},
+		AuthWhiteListedNames:      []string{},
+		AuthAdminRole:             "",
+		HTTPReadTimeout:           time.Duration(1) * time.Minute,
```

Comment:
For consistency, should we choose the same unit and apply it to all config?

Reply:
I think this dovetails with your previous comment. For consistency we'd need to go with the smallest unit we likely to encounter, which is millisecond, and express a minute as time.Duration(60000) * time.Millisecond, which is arguably more difficult to read than time.Duration(1) * time.Minute. I'm leaning towards the easiest to read - the smallest value with the right units factor.

Reply:
Per my other comment, as long as the CLI allows us to specify a duration I am satisfied leaving these as-is